packages/utils/src/lib/reports/md/list.ts

```diff
@@ -7,3 +9,9 @@ export function li(text: string, order: Order = 'unordered'): string {
   const style = order === 'unordered' ? '-' : '- [ ]';
   return `${style} ${text}`;
 }
+
+// eslint-disable-next-line no-magic-numbers
+export type Indentations = 1 | 2 | 3;
+export function indentation(text: string, level: Indentations = 1) {
+  return `${TAB.repeat(level)} ${text}`;
```

Comment:
Let's avoid using tab character, mixing with spaces can easily result in broken layouts.

```suggestion
  return `${'  '.repeat(level)} ${text}`;
```

Reply:
done